.travis.yml

```diff
@@ -4,3 +4,4 @@ node_js:
   - "0.12"
   - "4"
   - "5"
+  - "6"
```

Comment:
Good idea. Could you also remove Node 5 while you're in here? I don't think we need to test against Node 5 since it's not a [long term support](https://github.com/nodejs/LTS) release.